src/ispc.cpp

```diff
@@ -947,6 +948,46 @@ Target::Target(Arch arch, const char *cpu, ISPCTarget ispc_target, bool pic, boo
         this->m_maskBitCount = 32;
         CPUfromISA = CPU_Nehalem;
         break;
+    case ISPCTarget::sse41_i8x16:
```

Comment:
That about bundling 4.1 and 4.2 cases and have conditional assignment for `CPUfromISA`